src/main/java/sx/blah/discord/modules/ModuleLoader.java

```diff
@@ -356,4 +271,74 @@ private static File findFileForClass(List<File> files, String clazz) {
 	public static void addModuleClass(Class<? extends IModule> clazz) {
 		modules.add(clazz);
 	}
+
+	/**
+	 * Gets the modules loaded in this ModuleLoader instance.
+	 *
+	 * @return The list of loaded modules.
+	 */
+	public List<IModule> getLoadedModules() {
+		return loadedModules;
+	}
+
+	/**
+	 * Manually loads a module.
+	 *
+	 * @param module The module to load.
+	 * @return True if the module was successfully loaded, false if otherwise. Note: successful load != successfully enabled
+	 */
+	public boolean loadModule(IModule module) {
+		Class<? extends IModule> clazz = module.getClass(); if (clazz.isAnnotationPresent(Requires.class)) {
+			Requires annotation = clazz.getAnnotation(Requires.class);
+			if (!hasDependency(loadedModules, annotation.value())) {
+				return false;
+			}
+		} boolean enabled = module.enable(client); if (enabled) {
+			client.getDispatcher().registerListener(module);
+			if (!loadedModules.contains(module)) loadedModules.add(module);
```

Comment:
Inconsistency with single-line if statements being on the second line. (Not a big deal)